### PR TITLE
Api for Holiday controller

### PR DESCRIPTION
**Pull request by ror-uday**

### CODE REVIEW

File: spec/v1/holidays_spec.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe Api::V1::HolidaysController, type: :controller do
+  let!(:holiday) { FactoryBot.create(:holiday) }
+
+  describe 'GET# holidays#index' do
```
Comment (rorpyll) [inline spec/v1/holidays_spec.rb:8]: describe statement is not right

File: spec/v1/holidays_spec.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    before do
+      get :index
+    end
+
+    it 'index the holidays list' do
```
Comment (rorpyll) [inline spec/v1/holidays_spec.rb:13]: it statement is not right

**Review by rorpyll — CHANGES_REQUESTED** [review]

File: spec/v1/holidays_spec.rb (hunk 1 above)
Comment (ror-uday) [reply spec/v1/holidays_spec.rb]: changes done

File: spec/v1/holidays_spec.rb (hunk 2 above)
Comment (ror-uday) [reply spec/v1/holidays_spec.rb]: changes done

**Review by rorpyll — APPROVED** [review]

File: app/controllers/api/v1/holidays_controller.rb [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        }
+      end
+
+      private
+
+      def serializer
+        Api::V1::HolidaySerializer
+      end
+
+      def holiday_params
```
Comment (rorpyll) [inline app/controllers/api/v1/holidays_controller.rb:23]: why?

Comment (rorpyll) [inline app/controllers/api/v1/holidays_controller.rb:23]: why?

**Review by rorpyll — APPROVED** [review]

**Pull request closed without merging**